cmd/errtrace/main_test.go

```diff
@@ -440,6 +441,67 @@ func _() {
 	}
 }
 
+func TestStdinNoInputMessage(t *testing.T) {
+	// Verify that if there's no input on implicit stdin,
```

Comment:
if we expect different behaviour based on explicit or implicit, can we also test that

Reply:
Yep, done.